deploy/templates/default-azuredeploy-webjobs.json

```diff
@@ -0,0 +1,604 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ServiceName": {
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 20,
+      "metadata": {
+        "description": "Name for the service(s) being deployed.  Name will applied to all relevant services being created."
+      }
+    },
+    "RepositoryUrl": {
+      "type": "string",
+      "defaultValue": "https://github.com/Microsoft/iomt-fhir",
+      "metadata": {
+        "description": "Repository to pull source code from. If blank, source code will not be deployed."
+      }
+    },
+    "RepositoryBranch": {
+      "type": "string",
+      "defaultValue": "master",
+      "metadata": {
+        "description": "Source code branch to deploy."
+      }
+    },
+    "JobWindowUnit": {
+      "type": "string",
+      "allowedValues": [
+        "SECOND",
+        "MINUTE",
+        "HOUR"
+      ],
+      "metadata": {
+        "description": "The time period to collect events before sending them to the FHIR server."
+      },
+      "defaultValue": "MINUTE"
+    },
+    "JobWindowMagnitude": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 60,
+      "metadata": {
+        "description": "The magnitude of time period to collect events before sending them to the FHIR server."
+      },
+      "defaultValue": 5
+    },
+    "ThroughputUnits": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 20,
+      "metadata": {
+        "description": "The throughput units reserved for the Event Hubs created."
+      },
+      "defaultValue": 1
+    },
+    "AppServicePlanSku": {
+      "type": "string",
+      "allowedValues": [
+        "F1",
+        "D1",
+        "B1",
+        "B2",
+        "B3",
+        "S1",
+        "S2",
+        "S3",
+        "P1",
+        "P2",
+        "P3",
+        "P4"
+      ],
+      "defaultValue": "S1",
+      "metadata": {
+        "description": "The app service plan tier to use for hosting the required Azure Functions."
+      }
+    },
+    "ResourceLocation": {
```

Comment:
Since Stream Analytics is not used, we can probably expand the list of allowed locations. The only dependency that might restict the region is the App Service Plan. https://azure.microsoft.com/en-us/global-infrastructure/services/?products=app-service&regions=all